cpp/arcticdb/processing/aggregation.cpp

```diff
@@ -72,14 +73,6 @@ namespace
         }
     }
 
-    inline util::BitMagic::enumerator::value_type deref(util::BitMagic::enumerator iter) {
-        return *iter;
-    }
-
-    inline std::size_t deref(std::size_t index) {
```

Comment:
Maybe deleting unused functions is not always ideal? If they could be useful in the future it would be unfortunate to have to rewrite them.

Doesn't matter so much for these small functions, so I don't have a strong opinion on this.